.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: objective-c
+osx_image: xcode7.3
+before_install:
+  - export SWIFT_VERSION=swift-DEVELOPMENT-SNAPSHOT-2016-03-01-a
+  - curl -O https://swift.org/builds/development/xcode/$SWIFT_VERSION/$SWIFT_VERSION-osx.pkg
+  - sudo installer -pkg $SWIFT_VERSION-osx.pkg -target /
+  - export TOOLCHAINS=swift
+script:
+  - swift build
+  - .build/debug/SnowManDictionarySpecs
+  - .build/debug/SnowManLogicSpecs
```

Comment:
I merged `travis-setup` with master and then did a `git merge master` from this branch to add the changes. I am guessing from your tone that this is not correct :-1: